src/tools/plan-generator.ts

```diff
@@ -19,6 +19,7 @@ import {Ttl, TtlUnits} from '../runtime/recipe/ttl.js';
 import {Random} from '../runtime/random.js';
 import {findLongRunningArcId} from './storage-key-recipe-resolver.js';
 import {Capabilities} from '../runtime/capabilities.js';
+import {SchemaGraph} from './schema2graph.js';
```

Comment:
Not needed?

Reply:
Oops, leftover from earlier revision, removing.